docs/connector-development/tutorials/build-a-connector-the-hard-way.md

```diff
@@ -836,7 +836,7 @@ A full connector in less than 200 lines of code. Not bad! We're now ready to pac
 
 Our connector is very lightweight, so the Dockerfile needed to run it is very light as well. We edit the autogenerated `Dockerfile` so that its contents are as followed:
 
-```text
+```Dockerfile
```

Comment:
FWIW if gitbook doesn't show know what `Dockerfile` is supposed to do it might format it completely incorrectly. This isn't displayed on gitbook so no issues for now.

Reply:
yay markdown. at the very least vscode and github both display this correctly, so 🤞